code/modules/clothing/gloves/colored_gloves.dm

```diff
@@ -232,3 +215,4 @@
 	icon_state = "furglovesico"
 	item_state = "furgloves"
 	transfer_prints = TRUE
+	dyeable = FALSE
```

Comment:
another minor nitpick, but all items that can't be dyed anymore in the washing machine should be mentioned in the changelog and PR

Reply:
Will do. I know that fur gloves specifically were blacklisted before anyway.